KH2: Update Game Docs (#2188)
ArchipelagoMW/Archipelago: worlds/kh2/docs/setup_en.md

Reviewer: Wording feels weird, how about "when playing in an async or disconnecting from the server to play a different seed"

Have any questions on what's in logic? This spreadsheet has the answer [Requirements/logic sheet](https://docs.google.com/spreadsheets/d/1Embae0t7pIrbzvX-NRywk7bTHHEvuFzzQBUUpSUL7Ak/edit?usp=sharing)
<h2 style="text-transform:none";>F.A.Q.</h2>

- Why is my HP/MP continuously increasing without stopping?

>     - You do not have `JaredWeakStrike/APCompanion` setup correctly. Make Sure it is above the GOA in the mod manager.

Reviewer 2: ```suggestion
    - You do not have `JaredWeakStrike/APCompanion` setup correctly. Make sure it is above GOA in the mod manager.
```

- Why is my HP/MP continuously increasing without stopping when I have the APCompanion Mod?
    - You have a leftover GOA Script in your `Documents\KINGDOM HEARTS HD 1.5+2.5 ReMIX\scripts\KH2`.
- Why am I missing worlds/portals in the GoA?

>     - You are missing the required visit locking item to access the world/portal.

Reviewer 2: ```suggestion
    - You are missing the required visit-locking item to access the world/portal.
```

> - Why did I not load in to the correct visit

Reviewer 2: ```suggestion
- Why did I not load into the correct visit
```

> - Why did I not load into the correct visit

Reviewer 2: ```suggestion
- Why did I not load into the correct visit?
```

>     - You need to trigger a cutscene or visit The World That Never Was for it to update you have recevied the item.

Reviewer 2: ```suggestion
    - You need to trigger a cutscene or visit The World That Never Was for it to register that you have received the item.
```

- What versions of Kingdom Hearts 2 are supported?

>     - Currently `only` the most up to date version on the Epic Game Store is supported `1.0.0.8_WW`.

Reviewer 2: ```suggestion
    - Currently `only` the most up to date version on the Epic Game Store is supported: version `1.0.0.8_WW`.
```
